.github/workflows/long-running-azure.yaml

```diff
@@ -44,6 +44,7 @@ permissions:
   packages: write # Required for uploading the package
 
 on:
+  workflow_dispatch:
```

Comment:
Could you add a comment on when is it ok to manually trigger this workflow? Anything to keep in mind?

Reply:
added the comment